demos/python_demos/formula_recognition_demo/formula_recognition_demo.py

```diff
@@ -100,10 +162,16 @@ def build_argparser():
     args.add_argument("--max_formula_len",
                       help="Optional. Defines maximum length of the formula (number of tokens to decode)",
                       default="128", type=int)
+    args.add_argument("--conf_thresh", help="Optional. Probability threshold to trat model prediction as meaningful",
+                      default=CONFIDENCE_THRESH, type=float)
     args.add_argument("-d", "--device",
                       help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
                            "acceptable. Sample will look for a suitable plugin for device specified. Default value is CPU",
                       default="CPU", type=str)
+    args.add_argument("--camera_device", default=0, type=int,
+                      help='Optional. Device id of the web-camera. Change it only if you have more then one camera')
+    args.add_argument("--resolution", default=DEFAULT_RESOLUTION, type=int, nargs=2,
```

Comment:
```suggestion
    args.add_argument("--resolution", default=(1280, 720), type=int, nargs=2,
```
After removing default value from `InteractiveDemo`'s init you won't need `DEFAULT_RESOLUTION` const at all

Reply:
Deleted this constant